app/models/feature_geo_code.rb

```diff
@@ -16,6 +16,8 @@ class FeatureGeoCode < ActiveRecord::Base
   belongs_to :feature
   belongs_to :geo_code_type
   has_many :imports, :as => 'item', :dependent => :destroy
+
+  validates :geo_code_type_id, :feature_id, :geo_code_value, presence: true
```

Comment:
only leave geo_code_value in "validates". others are getting validated by belongs_to

Reply:
Fixed